services/utils/config/src/main/resources/ditto-akka-config.conf

```diff
@@ -102,7 +102,6 @@ akka {
       # Serialize Jsonifiable events with custom JSON serializer:
       "org.eclipse.ditto.model.base.json.Jsonifiable" = cbor
       "org.eclipse.ditto.model.base.exceptions.DittoRuntimeException" = cbor
-      "org.eclipse.ditto.signals.commands.devops.DevOpsCommandResponse" = json # to ensure readability
```

Comment:
@Yannic92 you´ll want to maintain this line as developers or applications using DevOpsCommands will otherwise receive CBOR responses to their commands.
This would make interacting with a running system a lot more difficult and will cause issues with internal tools that rely on the response being JSON.

Reply:
@erikescher will they? 
DevOpsCommand responses are always handled at HTTP level via gateway's `AbstractHttpRequestActor` which should use the JSON representation anyway, no matter how the message was serialized and deserialized in the cluster.

Reply:
I also verified this by setting it to "cbor". Still receiving well formed JSON at the HTTP API.